packages/types/src/interfaces/assetConversion/runtime.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2023 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { DefinitionsCall } from '../../types/index.js';
+
+export const runtime: DefinitionsCall = {
+  AssetConversionApi: [
+    {
+      methods: {
+        get_reserves: {
+          description: 'Get pool reserves',
+          params: [
+            {
+              name: 'asset1',
+              type: 'MultiAssetId'
```

Comment:
@jsidorenko  
I think, primary, we use asset-conversion on AssetHubs, so lets make them work at least,
so I would change it here to `Box<XcmV3MultiLocation>`  (please, try it locally, should work),

and then lets solve: https://github.com/paritytech/cumulus/issues/2915
and then lets find solution how to dynamically support: MultiAssetId (substrate node) vs MultiLocation (asset-hub),
because this could require some dynamic loading or some switch for node kind (substrate -> MultiAssetId, asset-hubs -> MultiLocation)

Reply:
Just tried, the `Box<XcmV3MultiLocation>` works well